firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteSchema.java

```diff
@@ -336,24 +336,37 @@ private void createV1RemoteDocumentCache() {
         });
   }
 
-  // TODO(indexing): Put the schema version in this method name.
-  private void createLocalDocumentsCollectionIndex() {
+  /**
+   * Creates the necessary tables to support document indexing.
+   *
+   * <p>The `index_configuration` table holds the configuration for all indices. Entries in this
+   * table apply for all users. It is not possible to only enable indices for a subset of users.
+   *
+   * <p>The `index_entries` table holds the index values themselves. An index value is created for
+   * every indexed field. If there are pending mutations that affect an indexed field, an additional
```

Comment:
This is not technically true is it? This suggests if there is a composite index with 2 fields, there will be two entries in this table.

How about `An entry is created for each value of the indexed field(s)...`

Reply:
You are correct. I changed it to say " An index value is created for each field combination that matches a configured index."

Thanks for catching.